guides/common/modules/proc_configuring-the-discovery-service.adoc

```diff
@@ -37,27 +37,41 @@ endif::[]
 hosts or virtual machines can be created with incorrect network configurations. On Dell servers, the new naming scheme can be disabled via `biosdevname=1` kernel command line option, for other hardware or virtual machines, the new naming scheme can be completely turned off via `net.ifnames=1`.
 endif::[]
 
-To use {ProjectServer} to provide the Discovery image, install the following RPM packages:
-
-* `tfm-rubygem-foreman_discovery`
-ifdef::satellite[]
-* `foreman-discovery-image`
-endif::[]
-* `tfm-rubygem-smart_proxy_discovery`
-
-The `tfm-rubygem-foreman_discovery` package contains the {Project} plug-in to handle discovered nodes, connections, and necessary database structures, and API.
-
+.Procedure
+. Install Discovery on {ProjectServer}:
++
 ifdef::satellite[]
 The `foreman-discovery-image` package installs the Discovery ISO to the `/usr/share/foreman-discovery-image/` directory.
 You can build a PXE boot image from this ISO using the `livecd-iso-to-pxeboot` tool.
 The tool saves this PXE boot image in the `/var/lib/tftpboot/boot` directory.
 For more information, see xref:Building_a_Discovery_Image_{context}[].
 endif::[]
-
-The `tfm-rubygem-smart_proxy_discovery` package configures {SmartProxyServer}, such as the integrated {SmartProxy} of {ProjectServer}, to act as a proxy for the Discovery service.
-
-ifndef::satellite[]
-You can use the `{foreman-installer}` tool to install the packages and the Discovery image.
++
+ifdef::satellite,orcharhino[]
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {foreman-installer} \
+--enable-foreman-plugin-discovery \
+--enable-foreman-proxy-plugin-discovery
```

Comment:
You're missing a backslash here and also below.
```suggestion
--enable-foreman-proxy-plugin-discovery \
```

Reply:
I removed it from Evgeni's [suggestion](https://github.com/theforeman/foreman-documentation/pull/1773#discussion_r1024033117)

Reply:
The current instruction is invalid because what you now wrote is actually 2 commands. First you suggest:
```
{foreman-installer} \
--enable-foreman-plugin-discovery \
--enable-foreman-proxy-plugin-discovery
```

And then
```
--foreman-proxy-plugin-discovery-install-images=false
```

This is because the shell interprets every line as a command, except when it ends with a `\`. Then it's seen as a continuation.

@evgeni suggested to drop `--foreman-proxy-plugin-discovery-install-images=false` (https://github.com/theforeman/foreman-documentation/pull/1773/files#r1023676046) and that's one way. Then you should remove the `\`. You now added back the argument, so it must be there.

So you should either drop the `...-images=...` argument (the best option) or add the backslash, but the current instruction must be changed.

Reply:
Agree and Addressed :+1: